lib/fs/basicfs_fileinfo_linuxish.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2022 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build aix || dragonfly || linux || openbsd || solaris || illumos
+// +build aix dragonfly linux openbsd solaris illumos
```

Comment:
And openbsd is more linuxish than bsdish - do you think we can withstand the shitstorm if die-hard-bsdler discovery this (never encountered them) :P

Reply:
I will challenge them to an office jousting competition